selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -669,16 +676,26 @@ public void goToCursorPositionVisible(int positionLine, int positionChar) {
     waitSpecifiedValueForLineAndChar(positionLine, positionChar);
   }
 
+  public void goToCursorPositionVisible(int editorIndex, int positionLine, int positionChar) {
```

Comment:
It duplicates `goToPosition(int editorIndex, int positionLine, int positionChar)` method.